### DISCUSSION TITLE
Webxr fallback behavior

### DISCUSSION THREAD

**Discussion by hd-esteban** [google/model-viewer#2311] [General]
What is the current behavior in case WebXR cannot be opened due to an error? 

Does it uses Scene viewer as fallback?

I have not been able to reproduce a use case where webxr fails but the device has Scene Viewer.

**Comment by elalish** [comment]
By default, yes, but you can control this behavior with the `ar-modes` attribute: https://modelviewer.dev/docs/#entrydocs-augmentedreality-attributes-arModes

To repro, clear your permissions for the site (or open an incognito tab), then reject the webXR permission when prompted. This will fail WebXR and you'll see the fallback behavior.
